database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/source/DBSource.java

```diff
@@ -66,6 +85,135 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     dbManager.validateJDBCPluginPipeline(pipelineConfigurer, getJDBCPluginId());
   }
 
+  class ListTablesRequest {
+    public String connectionString;
+    @Nullable
+    public String user;
+    @Nullable
+    public String password;
+    public String jdbcPluginName;
+    @Nullable
+    public String jdbcPluginType = "jdbc";
+  }
+
+  class GetSchemaRequest extends ListTablesRequest {
+    public String importQuery;
```

Comment:
this should not be called importQuery. We don't want somebody to confuse this with the importQuery that the source takes